server/configuration.go

```diff
@@ -23,6 +23,7 @@ import (
 // copy appropriate for your types.
 type configuration struct {
 	RejectPosts     bool
+	ExcludeBots     bool
```

Comment:
Nit pick: The should have the same ordering as `plugin.json`
```suggestion
	ExcludeBots     bool
	RejectPosts     bool
```

Reply:
Done, changed order and rebased :v: